data/Language/Verilog.yaml

```diff
@@ -153,15 +153,15 @@ keywords:
   - xnor
   - xor
 datatypes:
-  - shortint
-  - int
-  - longint
-  - byte
-  - bit
-  - logic
-  - reg
-  - integer
-  - real
-  - shortreal
-  - realtime
-  - time
+  - short_vlog
+  - int_c
+  - long_vlog
+  - byte_vlog
+  - bit_vlog
+  - logic_vlog
+  - reg_vlog
+  - integer_vlog
+  - real_vlog
+  - shortreal_vlog
+  - realtime_vlog
+  - time_vlog
```

Comment:
you havent created these datatype entries.

if you dont want to create them, leave the entry here as-is, and someone else can create the datatype files, and name them appropriately.